src/ops/ops.go

```diff
@@ -368,11 +369,13 @@ func (o *ops) renderDeploymentFiles(srcTemplate string, params map[string]interf
 	return nil
 }
 
-func (o *ops) GetVGByPV(pvName string) (string, error) {
+func (o *ops) GetVolumeGroupsByDisk(diskName string) ([]string, error) {
```

Comment:
maybe some test? Now we have option to create unitest for ops, i think we should add small test for this logic